### PR TITLE
apache-forrest: add Java dep, audit fixes

### PR DESCRIPTION
**Pull request by apjanke**
Addresses breakage in #44605

### CODE REVIEW

File: Library/Formula/apache-forrest.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class ApacheForrest < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=forrest/apache-forrest-0.9-sources.tar.gz"
   sha256 "c6ac758db2eb0d4d91bd1733bbbc2dec4fdb33603895c464bcb47a34490fb64d"
 
+  depends_on :java
```
Comment (DomT4) [inline Library/Formula/apache-forrest.rb]: Any idea if they want a specific minimum version?

Comment (apjanke) [inline Library/Formula/apache-forrest.rb]: Java 1.5; they're pretty basic. Given that (AFAIK) nothing pre-1.6 is available on OS X, do we need to note that?

Comment (DomT4) [inline Library/Formula/apache-forrest.rb]: Na. Wouldn't bother for 1.5.

Comment (apjanke) [inline Library/Formula/apache-forrest.rb]: All right. I think (imho) this is good to go as is if the brew-test-bot tests pass.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/653663d0a323871b6b54c0a0e1d7f7247a53c71d. Thank you for another contribution @apjanke! :star2:

**Pull request closed without merging**